### ISSUE TITLE
Suggesting new documentation

### ISSUE THREAD

**Issue by AdrielBento** [timgit/pg-boss#533]
Hi Tim, great job creating and maintaining this library. When I tried to use the library, I had some difficulty reading and interpreting the documentation. I think we could make some improvements to it.

Do you mind if I make a contribution to improving the documentation, switching to [VitePress](https://vitepress.dev/) among other improvements?

**Comment by timgit** [comment]
I don't mind docs improvements.  They are a work in progress, but I'd prefer this is deferred until after v11 is merged, since this would invalidate some of the changes

**Comment by michalczukm** [comment]
+1. I am happy to help there as well!

VitePress sounds like a good idea. 
I'd put one requirement there - **it should support search!**
- For VitePress https://vitepress.dev/reference/default-theme-search
- For Docusaurus https://docusaurus.io/docs/search#using-local-search

I used to docs being in the plain markdown - and just search via cmd+f on the page 😅  
Now I am confused where the pieces of docs live 😅

**Comment by AdrielBento** [comment]
No problem @timgit, I'll wait for the version 11 merge. Any prediction or estimate of when these merge will happen?

**Comment by timgit** [comment]
It just took 8 months, that's all ;)

**Comment by jmisur** [comment]
It would be fantastic if you improve the documentation. A bunch of use cases or examples explaining (or visualizing!) different send functions and their options would be invaluable. Right now it's impossible to be sure about the behavior without verifying your interpretation with manual experiments.

**Comment by timgit** [comment]
I agree!  I will see if I can recruit an agent to help me out with this.

**Comment by blue2cat** [comment]
@timgit, has any work been done toward documentation improvements? I'm interested in contributing.

**Comment by timgit** [comment]
Not yet.  Sounds good!
